### PR TITLE
Fix UnicodeDecodeError when decoding common_path_suffix

### PR DESCRIPTION
**Pull request by Horofic**
(DIS-1774)

### CODE REVIEW

**Comment by Horofic** [comment]
Fixes one of the issues in https://github.com/fox-it/dissect/issues/16

**Comment by yunzheng** [comment]
Would be nice to add a regression test

**Comment by pyrco** [comment]
I'm surprised the contents of these link structures are utf-8 encoded. Are you sure it's not local codepage or utf-16?

**Comment by Horofic** [comment]
@pyrco you are correct lnk files are local codepage encoded! The dissect.target `lnk` plugin is codepage aware, since it can get the used codepage from the `target`'s registry. The `parse-lnk` tool here (and lib) has to be codepage unaware, since (to my knowledge) there is no way of knowing what codepage to use.

**Comment by pyrco** [comment]
> @pyrco you are correct lnk files are local codepage encoded! The dissect.target `lnk` plugin is codepage aware, since it can get the used codepage from the `target`'s registry. The `parse-lnk` tool here (and lib) has to be codepage unaware, since (to my knowledge) there is no way of knowing what codepage to use.

Ah, it is just the tool, so that makes it less problematic. Although utf-8 will always be wrong. Maybe have a comment to that effect.  At some point we could add a codepage command line option or something if we decide to improve the tool.

File: dissect/shellitem/tools/lnk.py [hunk 1]
```diff
@@ -30,14 +29,12 @@ def parse(path: Path):
         lnk_iconlocation = lnk_file.stringdata.icon_location.string if lnk_file.flag("has_icon_location") else None
         lnk_arguments = lnk_file.stringdata.command_line_arguments.string if lnk_file.flag("has_arguments") else None
         local_base_path = (
-            lnk_file.linkinfo.local_base_path.decode("raw_unicode_escape")
+            lnk_file.linkinfo.local_base_path.decode(errors="ignore")
```
[the remark this answers is not in the document]
Comment (pyrco) [reply dissect/shellitem/tools/lnk.py:32]: I think that's the wrong PR

**Pull request merged**